server/network.go

```diff
@@ -252,7 +264,18 @@ func (lc *localNetwork) Start(ctx context.Context) error {
 	}
 
 	ux.Print(lc.log, logging.Blue.Wrap(logging.Bold.Wrap("create and run local network")))
-	nw, err := local.NewNetwork(lc.log, lc.cfg, lc.options.rootDataDir, lc.options.logRootDir, lc.options.snapshotsDir, lc.options.reassignPortsIfUsed, lc.options.redirectNodesOutput, lc.options.redirectNodesOutput, lc.options.walletPrivateKey)
+	nw, err := local.NewNetwork(
+		lc.log,
+		lc.cfg,
+		lc.options.rootDataDir,
+		lc.options.logRootDir,
+		lc.options.snapshotsDir,
+		lc.options.reassignPortsIfUsed,
+		lc.options.redirectNodesOutput,
+		lc.options.redirectNodesOutput,
+		lc.options.walletPrivateKey,
+		lc.options.genesisPath,
```

Comment:
I dont believe you need to pass it again, it is already in lc.cfg

Reply:
agree